multimodal-search-pdf/README.md

```diff
@@ -17,43 +17,20 @@
 
 </p>
 
-This example demonstrates how [Jina](http://www.jina.ai) can be used to search a repository of PDF files. The example employs a multimodal search architecture, allowing a user to query the data by providing text, or an image, or both simultaneously.
+This example demonstrates how [Jina](http://www.jina.ai) can be used to search a repository of PDF files.  
+The example employs a multimodal search architecture, allowing a user to query the data by providing text, or an image, or both simultaneously.
 
 What's included in this example:
 
-- Search text, image, PDF all in one Flow or in separate Flows
-- Leverage Jina Recursive Document Representation to segment and encode text
-- Speed up indexing time with parallel Peas
+- Search text, image, PDF all in one Flow 
+- Leverage Jina Recursive Document Representation to segment and encode text up to chunk-chunk level.
 - Use customized executors to better fit your needs
-- Provide detailed docstrings for YAML files to help you understand the example
-
-- [Table of contents](#pdf-search-with-jina)
-  * [Data preparation](#data-preparation)
-  * [Install](#install)
-  * [Run](#run)
-  * [Start the Server](#start-the-server)
-  * [Query via REST API](#query-via-rest-api)
-  * [Documentation](#documentation)
-  * [Community](#community)
-  * [License](#license)
-    
-<details>
- <summary>How to clone this repo❓</summary>
- 
- Because this repo contains many [more example projects](https://github.com/jina-ai/examples/), if you want to clone only this example, you can do this as following
- 
- ```bash
- git clone --depth 1 --filter=blob:none --sparse https://github.com/jina-ai/examples
- cd examples
- git sparse-checkout set multimodal-search-pdf
- ```
-</details>
-
-----
 
 ## Data preparation
 
-We have included several PDF blogs as toy data in [`toy_data`](toy_data). This data is ready to use straight away. You can replace this toy data with your own by simply adding new files to the [`toy_data`](toy_data) folder. Be careful to check that the files are supported by [`pdfplumber`](https://github.com/jsvine/pdfplumber).
+We have included several PDF blogs as toy data in [`toy_data`](toy_data).
+This data is ready to use straight away. You can replace this toy data with your own by simply adding new files to the [`toy_data`](toy_data) folder. 
+Be careful to check that the files are supported by [`pdfplumber`](https://github.com/jsvine/pdfplumber).
```

Comment:
Can you please add ```pip install wheel``` to the installation instruction before the requirements.txt, otherwise the pip installation will fail.

Reply:
https://github.com/jina-ai/examples/pull/629/commits/682e3ac8ab48268af5b2e3fcd91556641fd51605